app/sagas/state.js

```diff
@@ -32,25 +35,18 @@ const appHasComeBackToBackground = function* appHasComeBackToBackground() {
 		return;
 	}
 	try {
-		return yield RocketChat.setUserPresenceAway();
+		yield RocketChat.setUserPresenceAway();
+
+		const server = yield select(state => state.server.server);
+		yield saveLastLocalAuthenticationSession(server);
```

Comment:
Do this before `setUserPresenceAway`, because if you have a poor connection it'll be hung on `setUserPresenceAway`.